cea/interfaces/dashboard/plots/static/js/dashboard-plots.js

```diff
@@ -7,12 +7,12 @@ $(document).ready(function() {
     console.log("assigned categories!");
     console.log(categories);
 
-    $("#cea-plot-category").on("change", function(e){
+    $("[id|=cea-plot-category]").on("change", function(e){
```

Comment:
hey, cool! new syntax! i've never seen this before, what's it called?

Reply:
Oh its a jQuery selector for element attributes that matches element attribute values starting with the string followed by a hyphen. Was using something that required the same logic.